src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3837,6 +3837,63 @@ public LazyTryStatement(Lazy<IBlockStatement> body, Lazy<ImmutableArray<ICatchCl
     }
 
     /// <summary>
+    /// Represents a tuple expression.
+    /// </summary>
+    internal abstract partial class BaseTupleExpression : Operation, ITupleExpression
+    {
+        protected BaseTupleExpression(bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+                    base(OperationKind.TupleExpression, isInvalid, syntax, type, constantValue)
+        {
+        }
+        /// <summary>
+        /// Elements for tuple expression.
+        /// </summary>
+        public abstract ImmutableArray<IOperation> Elements { get; }
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitTupleExpression(this);
+        }
+        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
+        {
+            return visitor.VisitTupleExpression(this, argument);
+        }
+    }
+
+    /// <summary>
+    /// Represents a tuple expression.
+    /// </summary>
+    internal sealed partial class TupleExpression : BaseTupleExpression, ITupleExpression
+    {
+        public TupleExpression(ImmutableArray<IOperation> elements, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+            base(isInvalid, syntax, type, constantValue)
+        {
+            Elements = elements;
+        }
+        /// <summary>
+        /// Elements for tuple expression.
+        /// </summary>
+        public override ImmutableArray<IOperation> Elements { get; }
+    }
+
+    /// <summary>
+    /// Represents a C# try or a VB Try statement.
+    /// </summary>
+    internal sealed partial class LazyTupleExpression : BaseTupleExpression, ITupleExpression
+    {
+        private readonly Lazy<ImmutableArray<IOperation>> _lazyElements;
+
+        public LazyTupleExpression(Lazy<ImmutableArray<IOperation>> elements, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
```

Comment:
Note that the type may be `null`. Consider naming this field `typeOpt` if that is allowed by the generation tool.